src/java.base/share/classes/java/lang/reflect/ClassFileFormatVersion.java

```diff
@@ -316,65 +337,97 @@ public enum ClassFileFormatVersion {
 
     /**
      * The version introduced by the Java Platform, Standard Edition
      * 22.
      *
-     * @since 22
-     *
      * @see <a
      * href="https://docs.oracle.com/javase/specs/jvms/se22/html/index.html">
      * <cite>The Java Virtual Machine Specification, Java SE 22 Edition</cite></a>
+     * @since 22
      */
     RELEASE_22(66),
 
     /**
      * The version introduced by the Java Platform, Standard Edition
      * 23.
      *
-     * @since 23
-     *
      * @see <a
      * href="https://docs.oracle.com/javase/specs/jvms/se23/html/index.html">
      * <cite>The Java Virtual Machine Specification, Java SE 23 Edition</cite></a>
+     * @since 23
      */
     RELEASE_23(67),
 
     /**
      * The version introduced by the Java Platform, Standard Edition
      * 24.
      *
-     * @since 24
-     *
      * @see <a
      * href="https://docs.oracle.com/javase/specs/jvms/se24/html/index.html">
      * <cite>The Java Virtual Machine Specification, Java SE 24 Edition</cite></a>
+     * @since 24
      */
     RELEASE_24(68),
 
     /**
      * The version introduced by the Java Platform, Standard Edition
      * 25.
      *
-     * @since 25
-     *
      * @see <a
      * href="https://docs.oracle.com/javase/specs/jvms/se25/html/index.html">
      * <cite>The Java Virtual Machine Specification, Java SE 25 Edition</cite></a>
+     * @since 25
      */
     RELEASE_25(69),
-    ; // Reduce code churn when appending new constants
 
-    // Note to maintainers: when adding constants for newer releases,
-    // the implementation of latest() must be updated too.
+    // Note to maintainers: Add new constants right above.
+    // The implementation of latest() must be updated too.
+    /**
+     * An enum constant representing all preview VM features of the {@linkplain
+     * #latest() current Java SE release} in addition to those of the latest
+     * class file format version.  Unlike VM features associated to enum
+     * constants representing a class file format version, VM features
+     * associated to this enum constant are not {@linkplain #isSupported()
+     * supported} by later Java SE releases.
+     * <p>
+     * {@code class} files using any preview feature from the current Java SE
+     * release uses the same major version from that release, but uses the minor
+     * version {@value %04x ClassFile#PREVIEW_MINOR_VERSION} with all bits set
+     * to {@code 1}.  This Java Runtime Environment does not load any {@code
+     * class} file using preview features from other Java SE releases.
+     *
+     * @apiNote
+     * While this is not a class file format version, it can be considered as
+     * the class file format version of an arbitrary future Java SE release.
+     * Programmers should test their programs with preview features enabled to
+     * ensure the program is compatible with future Java SE releases.
+     * <p>
+     * This is a reflective preview API to allows tools running in Java runtime
```

Comment:
```suggestion
     * This is a reflective preview API that allows tools running in Java runtime
```

Reply:
Don't know why you recommend this, but this aims to explain why this API is reflective preview API instead of an essential preview API (that does not have `reflective = true`). Particular tools wish to perform this specific check, like `javap`; otherwise, I would have marked this as non-reflective, and tools must resort to a third-party knowledge pool to parse preview features related to the current release.

Reply:
It sounded wrong, maybe "to allow" (without 's') would be better.

Reply:
```suggestion
     * This is a reflective preview API to allow tools running in Java runtime
```